frontend/src/components/form/FormFields/CheckBox.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import QuestionFrom from './QuestionForm';
+import CheckBoxIndex from './CheckBoxIndex';
+
+function CheckBox() {
+
+    //temp state untill validation is done
+    
+    const [choices, setChoices] = useState([]);
+    
+     function addChoice() {
```

Comment:
this function has an extra space. I think we should have test for styling. cause I also tend to use 2 spaces for tabs and notice most people use 4.